src/game/boe.main.cpp

```diff
@@ -41,9 +47,10 @@ bool first_sound_played = false,spell_forced = false;
 bool party_in_memory = false;
 std::shared_ptr<cScrollbar> text_sbar, item_sbar, shop_sbar;
 std::shared_ptr<cButton> done_btn, help_btn;
-rectangle sbar_rect = {283,546,421,562};
-rectangle shop_sbar_rect = {67,258,357,274};
-rectangle item_sbar_rect = {146,546,253,562};
+// XXX move these 3 to boe.ui.cpp ?
```

Comment:
Please use either `TODO:` or `FIXME:` for todo-comments, instead of `XXX`.

Reply:
done